docs/en/observability/quickstarts/collect-data-with-aws-firehose.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[collect-data-with-aws-firehose]]
+= Quickstart: Collect data with AWS Firehose
+
+preview::[]
+
+In this quickstart guide, you'll learn how to send logs and metrics to Elastic using AWS Firehose.
+
+The AWS Firehose receiver is based on a CloudFormation template that can collect all available CloudWatch logs and metrics for your AWS account.
```

Comment:
> The AWS Firehose receiver is based on a CloudFormation template

I'd suggest changing this to something like "The AWS Firehose streams are created using CloudFormation template ...", this better reflects what's going on